### DISCUSSION TITLE
JavaScript inside an e-mail - Question / Example of it (Outlook)

### DISCUSSION THREAD

**Discussion by ProfessorManhattan** [mjmlio/mjml#2655] [Q&A]
Hey, I was wondering how I received an e-mail from Outlook that looked like it has JS inside of it. Not sure if this feature is unique to Outlook. Does anybody know how this is possible?

Short video demonstrating what I saw: https://www.youtube.com/shorts/9dpRZaoRupQ

**Comment by iRyusa** [comment]
Most likely unique to Yammer + Outlook as Yammer is in Microsoft ecosystem.Not likely something you can replicateOn 22 Mar 2023, at 20:25, Brian Zalewski ***@***.***> wrote:﻿
Hey, I was wondering how I received an e-mail from Outlook that looked like it has JS inside of it. Not sure if this feature is unique to Outlook. Does anybody know how this is possible?
Short video demonstrating what I saw: https://www.youtube.com/shorts/9dpRZaoRupQ

**Comment by ProfessorManhattan** [comment]
Well if Outlook supports it.. why not? Wondering if it's a special
Microsoft trusted partners thing or do I have to worry even more about
opening emails in Outlook

**Comment by iRyusa** [comment]
As said yammer is owned by Microsoft so it might be only a special integration between both product and not something available publiclyOn 23 Mar 2023, at 00:02, Brian Zalewski ***@***.***> wrote:﻿
Well if Outlook supports it.. why not? Wondering if it's a special
Microsoft trusted partners thing or do I have to worry even more about
opening emails in Outlook
